presto-server-rpm/src/main/rpm/preremove

```diff
@@ -0,0 +1,6 @@
+# Pre erase script
+
+if ! /etc/init.d/presto status | grep -q 'Not running'; then 
+    echo "Please stop the Presto service before erasing RPM"
```

Comment:
Is it what other RPMs do? eg. postgres?

Reply:
I do not know, I checked that once but I have found anything interesting.